### DISCUSSION TITLE
Which commit should I use as a casual, but fearless user?

### DISCUSSION THREAD

**Discussion by jonaseberle** [Ortho4XP/Ortho4XP#56] [Q&A]
Hi, so far I have been using the tag https://github.com/oscarpilote/Ortho4XP/releases/tag/v1.31 and it has been very stable.

I am usually ok to use `master`/`main` in projects to be more close to current development and be able to file meaningful bug reports.
I am a software developer, but have not done much with python and will most likely not be able to fix bugs myself.

What do you reckon? What should I use? Are you in the middle of some huge refactoring or does it make sense to test and use `master`?
Or would you tag some newer dev release for us more casuals?

**Comment by d41k4n** [comment]
Ortho4XP is not really well maintained, I'm afraid. I think currently most of us are waiting for an updated version for XP12 by its original author @oscarpilote as was alluded in [this post on forums.x-plane.org](https://forums.x-plane.org/index.php?/forums/topic/268914-updating-existing-ortho-for-xp12/&do=findComment&comment=2417062).

Besides, this fork is/was an attempt to resolve the [original Ortho4XP's](https://github.com/oscarpilote/Ortho4XP) accumulated technical debt and refactor its codebase to make it easier to install, test and maintain... However, the activity seems to have stalled here, too...

The v1.31 tag in the original repo was created by me in a desperate effort to highlight the changes made after @oscarpilote released the [Windows binary package for 1.30](https://drive.google.com/open?id=1PWgp33gDUQbY3uZmYbTi3ccMPLIJMQAQ) on October 18th, 2018...
